dashboard/test/models/regional_partner_test.rb

```diff
@@ -262,6 +262,8 @@ class RegionalPartnerTest < ActiveSupport::TestCase
       assert_equal WORKSHOP_APPLICATION_STATES[:currently_open], regional_partner.summer_workshops_application_state
       assert_equal "September 25, 2018", regional_partner.summer_workshops_earliest_apps_open_date
       assert_nil regional_partner.link_to_partner_application
+
+      assert_equal "October  2, 2018", regional_partner.upcoming_priority_deadline_date
```

Comment:
is the extra space in there intentional?

Reply:
Good question.  Yes, the use of the blank-padded day of the month (`%e`) goes back a couple years and we've just continued using it here.